### ISSUE TITLE
EKS Support Service IPv4 range

### ISSUE THREAD

**Issue by jwenz723** [hashicorp/terraform-provider-aws#15505]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add support in `aws_eks_cluster` for the new Service IPv4 range feature of EKS clusters: https://aws.amazon.com/about-aws/whats-new/2020/10/amazon-eks-supports-configurable-kubernetes-service-ip-address-range/

### New or Affected Resource(s)

* aws_eks_cluster

### Potential Terraform Configuration

```hcl
resource "aws_eks_cluster" "cluster" {
  name     = var.cluster-name
  serviceIpv4Cidr = "10.0.0.0/16" // something like this is what I would imagine
}
```

### References

I'm not aware of any other open issues for this feature.

**Comment by anGie44** [comment]
hi @jwenz723, thank you for creating this issue! To follow along the AWS API, just noting here that this new argument `service_ipv4_cidr` should also appear w/in it's parent argument `kubernetes_network_config`, with an updated potential config:
```
resource "aws_eks_cluster" "cluster" {
  name     = var.cluster-name
  kubernetes_network_config {
    service_ipv4_cidr = "10.0.0.0/16" // something like this is what I would imagine
}
```
Reference: https://docs.aws.amazon.com/eks/latest/APIReference/API_CreateCluster.html

**Comment by philnichol** [comment]
@anGie44 Mind if I have a crack at that?

**Comment by anGie44** [comment]
This feature has been merged and will release with `v3.19.0` of the terraform aws provider, likely out this Thursday.
